src/app/components/holdplan/DayRadio.module.scss

```diff
@@ -0,0 +1,66 @@
+@use "src/app/variables" as *;
+
+.container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $spacing-48;
+  height: $spacing-80;
+
+  .input {
+    position: absolute;
+    appearance: none;
+    width: 100%;
+    height: 100%;
+    background-color: $white;
+    border: solid $border-width $off-black400;
+    color: $black;
+
+    &:hover,
+    &:checked {
+      background-color: $off-black400;
+      border: none;
+    }
+    &:focus {
+      outline: none;
+      border: solid 3px #0e59e4;
+    }
+
+    &:hover + .label,
+    &:checked + .label {
+      color: $white;
+    }
+  }
+
+  .label {
+    position: absolute;
+    font-family: $poppins-font;
+    font-size: $spacing-16;
+    font-weight: $font-regular;
+    line-height: $line-height-m;
+    text-transform: uppercase;
+  }
+
+  .desk_text {
+    display: none; //skjuler desktop teksten i mobileview
+  }
+
+  .mobile_text {
+    display: inline;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    width: 150px;
```

Comment:
Det er ikke muligt, at bruge vores variabler hér? :-)

Reply:
Vi har ikke en variable der hedder 150px, men designet skal være 150px. Jeg ved vi har sagt vi runder op eller ned, men her er der ikke tale om en margin eller anden form for spacing, og det vil påvirke designet meget, hvis den skulle være 120 eller 160 :-)